src/Smidge/SmidgeHelper.cs

```diff
@@ -128,7 +128,7 @@ public async Task<HtmlString> CssHereAsync(PreProcessPipeline pipeline = null, b
             var urls = await GenerateCssUrlsAsync(pipeline, debug);
             foreach (var url in urls)
             {
-                result.AppendFormat("<link href='{0}' rel='stylesheet' type='text/css'/>", url);
```

Comment:
is this `type` attribute no obsolete too?

Reply:
Yes, you're right, as is the closing slash. Not sure what to do now. Do I need to cancel and resubmit the PR? I'm not a github expert, lol

Reply:
No all good, you've already removed the type attribute which is why I was asking.  But I see the modern way is this https://www.w3schools.com/tags/tag_link.asp. All you need to do is make this change in the branch that you created this PR from and push a new revision. It will automatically just update this PR :) does that make sense?

Reply:
As it happens, I'd already removed that "type" attribute. Don't know why it didn't appear in the original PR. I've pushed it again anyway. I "think" it's correct now.

Reply:
Looks good!